lib/Echidna/Config.hs

```diff
@@ -56,10 +57,16 @@ instance FromJSON EConfig where
                 fprefix <- v .:? "prefix"  .!= "echidna_"
                 let goal fname = if (fprefix <> "revert_") `isPrefixOf` fname then ResRevert else ResTrue
                 return $ TestConf (\fname -> (== goal fname)  . maybe ResOther classifyRes . view result) (const psender)
+
+        timeoutp :: Y.Parser Integer
+        timeoutp = v .:? "timeout" >>= \case (Nothing :: Maybe Integer) -> pure (-1) 
+                                             (Just n) -> pure (n*1e12)
```

Comment:
I think `timeoutp = ((*1e12) <$> v .:? "timeout") .!= -1` looks better here. Also I'm surprised the `\case` doesn't automatically type to `Maybe Integer` due to the `n*1e12`.